Terminal.Gui/Core.cs

```diff
@@ -1461,24 +1494,67 @@ public override string ToString ()
 			return $"{GetType ().Name}({Id})({Frame})";
 		}
 
+		/// <summary>
+		/// Specifies the event arguments for <see cref="MouseEvent"/>
+		/// </summary>
+		public class MouseEventEventArgs : EventArgs {
+			/// <summary>
+			/// Constructs.
+			/// </summary>
+			/// <param name="me"></param>
+			public MouseEventEventArgs (MouseEvent me) => mouseEvent = me;
+			/// <summary>
+			/// The <see cref="MouseEvent"/> for the event.
+			/// </summary>
+			public MouseEvent mouseEvent { get; set; }
```

Comment:
Public property names should be `CamelCase`. `mouseEvent` should be `MouseEvent`.

Reply:
I don't see any option to put a label but only to filter. Then you can guide me in this, please.